app/Http/Controllers/Api/CloudProviderTokensController.php

```
@@ -0,0 +1,535 @@
+
+        $cloudProviderToken = CloudProviderToken::create([
+            'team_id' => $teamId,
+            'provider' => $request->provider,
+            'token' => $request->token,
+            'name' => $request->name,
+        ]);
+
+        return response()->json([
+            'uuid' => $cloudProviderToken->uuid,
+        ])->setStatusCode(201);
+    }
```

PHPMD (2.15.0)

The method store() has a Cyclomatic Complexity of 13. The configured cyclomatic complexity threshold is 10. (undefined)

(CyclomaticComplexity)